test/unit/regapic.ts

```diff
@@ -107,36 +107,63 @@ describe('regapic', () => {
     });
   });
 
-  it('should support enum conversion in proto message response', done => {
-    const requestObject = {name: 'shelves/shelf-name'};
-    const responseObject = {
-      name: 'shelf-name',
-      theme: 'shelf-theme',
-      type: 1,
-    };
-    // incomplete types for nodeFetch, so...
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    sinon.stub(nodeFetch, 'Promise' as any).returns(
-      Promise.resolve({
-        ok: true,
-        arrayBuffer: () => {
-          return Promise.resolve(Buffer.from(JSON.stringify(responseObject)));
-        },
-      })
-    );
+  describe('should support enum conversion in proto message', () => {
+    it('should support enum conversion in proto message response', done => {
+      const requestObject = {name: 'shelves/shelf-name'};
+      const responseObject = {
+        name: 'shelf-name',
+        theme: 'shelf-theme',
+        type: 1,
+      };
+      // incomplete types for nodeFetch, so...
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      sinon.stub(nodeFetch, 'Promise' as any).returns(
+        Promise.resolve({
+          ok: true,
+          arrayBuffer: () => {
+            return Promise.resolve(Buffer.from(JSON.stringify(responseObject)));
+          },
+        })
+      );
+
+      gaxGrpc.createStub(libraryService, stubOptions).then(libStub => {
+        libStub.getShelf(
+          requestObject,
+          {},
+          {},
+          (err: {}, result: {name: {}; theme: {}; type: {}}) => {
+            assert.strictEqual(err, null);
+            assert.strictEqual('shelf-name', result.name);
+            assert.strictEqual('TYPEONE', result.type);
+            done();
+          }
+        );
+      });
+    });
 
-    gaxGrpc.createStub(libraryService, stubOptions).then(libStub => {
-      libStub.getShelf(
-        requestObject,
-        {},
-        {},
-        (err: {}, result: {name: {}; theme: {}; type: {}}) => {
+    it('should support enum conversion in proto message request', done => {
+      const shelf = {
+        name: 'shelf-name',
+        theme: 'shelf-theme',
+        type: 'TYPEONE',
```

Comment:
Can you add one more test that passes 1 in the request?  We do want to make sure we can accept both `TYPEONE` and 1 in the request object.  Other than that, LGTM!

Reply:
SG. Thanks for pointing out. 👍